Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -237,14 +237,24 @@ ErrorCode HardwareBreakpointManager::enableDebugCtrlReg(uint32_t &ctrlReg,
     }
   }
 
+  // Make sure to clear top half of the register
+  for (size_t i = 32; i < 64; ++i) {
```

Comment:
This is a lame way of doing this. You can use (or write, in `Bits.h`) proper bit masking functions.